src/ipc-handlers.ts

```diff
@@ -1409,3 +1409,7 @@ export async function handleNewSite( event: IpcMainInvokeEvent, newSite: NewSite
 		newSites: userData.newSites?.filter( ( s ) => s.id !== newSite.id ),
 	} );
 }
+
+export function comparePaths( event: IpcMainInvokeEvent, path1: string, path2: string ) {
```

Comment:
I am thinking - should we go with the same naming for internal and for IPC?
Or, mauyb e as option woudl be better to come up with some rule to follow inside our team, e.g. - if you want to expose some function, use the same name but with some prefix or something like that.
WDYT?
```suggestion
export function arePathsEqual( event: IpcMainInvokeEvent, path1: string, path2: string ) {
```

Reply:
I agree they would ideally use the same name, this was just my lazy choice to work around the problem of name collisions in `ipc-handlers.ts`. I hit "merge" _just_ before you submitted your review, so I would just leave this as is.

Reply:
Sure, let's leave it as is, I just wanted to raise this discussion for future cases